terraform/modules/hub/files/cloud-init/prometheus.sh

```diff
@@ -152,7 +152,8 @@ if [ -z "$(lsblk | grep "$vol" | awk '{print $7}')" ] ; then
     echo "volume /dev/$vol is mounted ; mounting"
 
   if grep -qv "/dev/$vol" /etc/fstab ; then
-    echo "/dev/$vol /srv/prometheus ext4 defaults,nofail 0 2" >> /etc/fstab
+    e2label /dev/$vol srv-prometheus
```

Comment:
Does this depend on having `/dev/nvme1n1` be the "right" volume the first time that we boot?

Would it perhaps be safer if we did some bash magic and find the name of the device based on its size?

Reply:
I assume it's always the right volume on first boot, it will presumably have formatted the first one (nvme0n1) as the root device.

Reply:
Certainly so far we haven't seen the problems on first boot; only on reboot

Reply:
That said, it'd be good to verify that Prometheus gets the big disk and / gets the little one